<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QWidget" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controler ASE</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="12,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="5,0,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Endereço RTSP câmera:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="rtsp_camera">
               <property name="text">
                <string>rtsp://admin:r0d0@192.168.110.9:554/cam/realmonitor?channel=7&amp;subtype=0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Nome da câmera:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLineEdit" name="nome_camera">
                 <property name="text">
                  <string>Camera 7</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="adicionar_but">
                 <property name="text">
                  <string>Adicionar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
           <item>
            <widget class="QGroupBox" name="cameras_group">
             <property name="title">
              <string>Cameras</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QVBoxLayout" name="cameras_layout"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="options_group">
             <property name="title">
              <string>Opções</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>IP WhatsApp Service</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ip_whatsapp">
                <property name="text">
                 <string>n/a</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Camera cadastro</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QLineEdit" name="ip_camera_cadastro">
                    <property name="text">
                     <string>rtsp://192.168.110.247:5554/back</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="device_n"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="isNetwork">
                  <property name="text">
                   <string>Network</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="novos_usuarios">
                <property name="text">
                 <string>Novos Usuários</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Intervalo deteção</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="detect_spin">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>99</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="enhance_check">
                <property name="text">
                 <string>Melhoramento de imagem</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="face_tracking_check">
                <property name="text">
                 <string>Face Tracking</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="color_check">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Detect w/ Color</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="show_detections">
                <property name="text">
                 <string>Show detections</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="skip_check">
                <property name="text">
                 <string>Enable skip</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Log de atividades</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="log"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Usuários sendo detectados: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lista_usuario"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
